src/modules/samples/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-add_subdirectory(cpp)
\ No newline at end of file
+add_subdirectory(cpp)
```

Comment:
We also need the sample to be published in pure C language. 

This will also allow us to compare 1:1 with Rust.